mx-semantics/main/blocks/configuration.md

```diff
@@ -0,0 +1,15 @@
+```k
+
+module MX-BLOCKS-CONFIGURATION
+    imports INT-SYNTAX
+
+    configuration
+        <mx-blocks>
+            <mx-current-block>
+                <mx-current-block-timestamp> 0 </mx-current-block-timestamp>
```

Comment:
Right now my understanding is that our timing information is arbitrary, as we do not compute time at the moment, correct?

Reply:
Yes, that's correct. However, we need to be able to simulate time for the lending contract, which uses `self.blockchain().get_block_timestamp()`. Arguably, maybe I shouldn't have implemented this hook now since our priority is the swap contract, but it was quite easy, so whatever.